testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusServerDeployableContainer.java

```diff
@@ -258,6 +258,9 @@ private ProcessBuilder getProcessBuilder() {
         }
 
         addStorageOptions(storeProvider, commands);
+        if (System.getProperty("auth.server.quarkus.log-level") != null) {
+            commands.add("--log-level=" + System.getProperty("auth.server.quarkus.log-level"));
+        }
```

Comment:
Why is this being added? We have built-in support for configuring logging for the Quarkus dist

Reply:
To have this support available in integration auth-server-quarkus. Without this addition, it is not possible to pass in the logging parameter into the tested server.